numpyro/infer/__init__.py

```diff
@@ -24,6 +24,7 @@
     'log_likelihood',
     'ELBO',
```

Comment:
Do you want to do the renaming in this PR?

Reply:
@neerajprad I'll do it in a separate PR. I just fixed an issue at `contrib/module.py` where params given to `optim.init` method has a different order w.r.t. params got from `optim.get_params` method. It is due to in JAX, `unflatten_tree(flatten_tree(x))` is the same dict as `x` but having keys sorted. If keys are not consistent, latent sites will be inconsistent after we lifted them to `sample` sites, which violate mean field assumption: latent sites in guide should be as the same order as latent sites in model.